### ISSUE TITLE
[Request] QSelect - clearable prop

### ISSUE THREAD

**Issue by Mohanrau** [quasarframework/quasar#1018]
### Software version

Quasar: 0.14.5
OS: windows 10
Node:
NPM:
Browsers:
iOS:
Android:
Any other software related to your bug:

### What were you expecting?
Add Clearable single q-select
Ref - http://element.eleme.io/#/en-US/component/select#clearable-single-select

**Comment by igorgo** [comment]
It simply to create your own component that extends `QSelect` with prop `clearable` and adds the existing property `after` with
```javascript
[{
  icon: 'cancel',
  content: true,
  handler: () => { this.$emit('input', this.multiple ? [] : '') }
}]
```
But I agree, this feature must be in the base component.

**Comment by rstoenescu** [comment]
Agreed. Will get into v0.14.7

**Comment by Mohanrau** [comment]
@igorgo thank you for quickfix @rstoenescu good to hear that thanks man

**Comment by rstoenescu** [comment]
Available in edge and future v0.14.7
